example/android/src/main/java/com/splendo/kaluga/example/bluetooth/BluetoothMoreActivity.kt

```diff
@@ -18,22 +18,21 @@
 package com.splendo.kaluga.example.bluetooth
 
 import android.os.Bundle
-import com.splendo.kaluga.architecture.navigation.toNavigationBundle
+import com.splendo.kaluga.architecture.navigation.NavigationBundleSpecType
+import com.splendo.kaluga.architecture.navigation.toTypedProperty
 import com.splendo.kaluga.architecture.viewmodel.KalugaViewModelActivity
+import com.splendo.kaluga.bluetooth.device.SerializableIdentifier
 import com.splendo.kaluga.example.databinding.ActivityBluetoothMoreBinding
 import com.splendo.kaluga.example.shared.viewmodel.bluetooth.BluetoothDeviceDetailViewModel
-import com.splendo.kaluga.example.shared.viewmodel.bluetooth.DeviceDetailsSpec
-import com.splendo.kaluga.example.shared.viewmodel.bluetooth.DeviceDetailsSpecRow
 import org.koin.androidx.viewmodel.ext.android.viewModel
 import org.koin.core.parameter.parametersOf
 
 class BluetoothMoreActivity : KalugaViewModelActivity<BluetoothDeviceDetailViewModel>() {
 
     override val viewModel: BluetoothDeviceDetailViewModel by viewModel {
-        val deviceDetailsSpec = DeviceDetailsSpec()
-        intent.extras?.toNavigationBundle(deviceDetailsSpec)?.let { bundle ->
-            parametersOf(bundle.get(DeviceDetailsSpecRow.UUIDRow))
-        } ?: parametersOf("")
+        parametersOf(
+            intent.extras?.toTypedProperty(NavigationBundleSpecType.SerializedType(SerializableIdentifier.serializer()))!!.identifier
```

Comment:
I see this recurring, is it not possible to implement a pattern for this in kaluga?

Reply:
https://github.com/splendo/kaluga/pull/624